src/games/GameSession.ts

```diff
@@ -93,8 +93,7 @@ export abstract class GameSession<T extends SessionConfig> {
       .map((user) => this.hub.guild.members.resolve(user.id));
   }
 
-  public async patchConfig(
-    partial: Partial<T>,
+  public async Configuration(
```

Comment:
Function names should start with a lowercase letter. This function doesn't have to be async. Also just configuration is quite vague, how about
```suggestion
  public setBaseConfiguration(
```
The return type would then just be `void`.